servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -4552,7 +4552,7 @@ public void putServiceIdentitySystemMeta(ResourceContext ctx, String domainName,
         verifyAuthorizedServiceOperation(principal.getAuthorizedService(), caller);
 
```

Comment:
Unlike the other methods, I didn't change the caller as the name of the method "putServiceIdentitySystemMeta" is different than the caller "putservicesystemmeta". Changing this will break the call to verifyAuthorizedServiceOperation as it appears differently in authorized_services.json file